package-lock.json

```diff
@@ -1,7179 +1,8 @@
 {
   "name": "ds-wagtail",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Does `package-lock.json` require to be updated for the Django upgrade? It also appears to be downgraded.

Reply:
These changes are all part of the poetry command, I am not sure I can answer this question. :)

Reply:
Please can you share the poetry command for django upgrade?

Reply:
I am not very sure , but I think it was poetry add  "django==4.0"